firebase-firestore/src/main/java/com/google/firebase/firestore/local/LruGarbageCollector.java

```diff
@@ -15,16 +15,82 @@
 package com.google.firebase.firestore.local;
 
 import android.util.SparseArray;
+import com.google.firebase.firestore.FirebaseFirestoreSettings;
 import com.google.firebase.firestore.core.ListenSequence;
+import com.google.firebase.firestore.util.Logger;
 import java.util.Comparator;
 import java.util.PriorityQueue;
 
 /** Implements the steps for LRU garbage collection. */
-class LruGarbageCollector {
+public class LruGarbageCollector {
+  public static class Params {
+    private static final long COLLECTION_DISABLED = FirebaseFirestoreSettings.CACHE_SIZE_UNLIMITED;
+    private static final long DEFAULT_CACHE_SIZE_BYTES = 100 * 1024 * 1024; // 100mb
```

Comment:
Can you refactor this so that you don't have to specify the default size twice?

It seems like you could get rid of `Params`. The two modes you have are: 
- A size is specified, either default or non-default. Your settings are cacheSizeBytes, 10, 1000
- GC is disabled. You don't need to instantiate the `LruGarbageCollector` at all.